TOOLS/umpv

```diff
@@ -85,17 +89,26 @@ def start_mpv(files: Iterable[str], socket_path: str) -> None:
     subprocess.Popen(mpv_command, start_new_session=True)
 
 def main() -> None:
-    files = (os.path.abspath(f) if not is_url(f) else f for f in sys.argv[1:])
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--replace",
```

Comment:
maybe instead `--replace` just allow string input with that would be inserted as a flag to insert? Would be more versatile.